en/setup/citationkeypatterns.md

```diff
@@ -18,57 +18,70 @@ The default key pattern is **`[auth][year]`**, and this could produce keys like
 
 #### Title-related key patterns
 
-* **`[shorttitle]`**: The first 3 words of the title, ignoring any function words \(see below\). For example, `An awesome paper on JabRef` becomes `AwesomePaperJabref`.
-* **`[veryshorttitle]`**: The first word of the title, ignoring any function words \(see below\). For example, `An awesome paper on JabRef` becomes `Awesome`.
-* **`[camel]`**: Capitalize and concatenate all the words of the title. For example, `An awesome paper on JabRef` becomes `AnAwesomePaperOnJabref`.
-* **`[title]`**:  Capitalize all the significant words of the title, and concatenate them. For example, `An awesome paper on JabRef` becomes `AnAwesomePaperonJabref`.
+* **`[shorttitle]`**: The first 3 words of the title, ignoring any function words \(see below\). For example, `An awesome paper on JabRef` becomes `AwesomePaperJabref`
+* **`[veryshorttitle]`**: The first word of the title, ignoring any function words \(see below\). For example, `An awesome paper on JabRef` becomes `Awesome`
+* **`[camel]`**: Capitalize and concatenate all the words of the title. For example, `An awesome paper on JabRef` becomes `AnAwesomePaperOnJabref`
+* **`[title]`**:  Capitalize all the significant words of the title, and concatenate them. For example, `An awesome paper on JabRef` becomes `AnAwesomePaperonJabref`
 
 JabRef considers the following words to be [function words](https://en.wikipedia.org/wiki/Function_word): "a", "an", "the", "above", "about", "across", "against", "along", "among", "around", "at", "before", "behind", "below", "beneath", "beside", "between", "beyond", "by", "down", "during", "except", "for", "from", "in", "inside", "into", "like", "near", "of", "off", "on", "onto", "since", "to", "toward", "through", "under", "until", "up", "upon", "with", "within", "without", "and", "but", "for", "nor", "or", "so", "yet".
 
 #### Other key patterns
 
 * **`[firstpage]`**: The number of the first page of the publication \(Caution: this will return the lowest number found in the pages field, i.e. for `7,41,73--97` it will return `7`.\)
-* **`[pageprefix]`**: The non-digit prefix of pages \(like "L" for L7\) or "" if no non-digit prefix exists \(like "" for `7,41,73--97`\) .
-* **`[keywordN]`**: Keyword number N from the “keywords” field, assuming keywords are separated by commas or semicolons.
+* **`[pageprefix]`**: The non-digit prefix of pages \(like "L" for L7\) or "" if no non-digit prefix exists \(like "" for `7,41,73--97`\) 
+* **`[keywordN]`**: Keyword number N from the “keywords” field, assuming keywords are separated by commas or semicolons
 * **`[lastpage]`**: The number of the last page of the publication \(See the remark on `firstpage`\)
 * **`[shortyear]`**: The last 2 digits of the publication year
 
+#### BibTeX fields
+
+In addition to the special field markers, most BibTeX, BibLaTeX, and JabRef field names can be used directly. If you regularly use a field marker not on this list, you are encouraged to add it.
```

Comment:
"biblatex" (without capitals)

Reply:
Hey mlep! I appreciate the feedback.
I think I need to read up a bit more before I answer these points, when you want me to include that example, are you referring to how `RegExpBasedFileFinder` is used and would you like it to be included in the help file?

Something analogous to having a help document exclusively for the bracketed expressions?

Reply:
Yupp, I believe you are right about "biblatex" based on https://github.com/plk/biblatex and their documentation. I was looking at https://ctan.org/pkg/biblatex?lang=en

Reply:
Yes, all lowercase, as per request of the author: https://github.com/JabRef/jabref/issues/2493